src/main/java/org/folio/bulkops/service/ElectronicAccessService.java

```diff
@@ -74,4 +74,10 @@ private ElectronicAccess restoreElectronicAccessItem(String access, String delim
     }
     return null;
   }
+
+  private String getRelationshipName(ElectronicAccess access) {
+    log.info("getRelationshipName {}, {}, {}", access.getRelationshipId(), access.getTenantId(), folioExecutionContext.getTenantId());
```

Comment:
Remove this logging or make it debug if you really need it.

Reply:
Changed to debug.